src/Rules/Resources/HowToFix.resx

```diff
@@ -478,6 +478,11 @@ Provide a string for the LocalizedControlType property that concisely describes
 
 Better:
 If possible, use a predefined (non-custom) control type and the default localized control type. The UIA framework will provide the correct localized control type automatically.</value>
+  </data>
+  <data name="LocalizedControlTypeNotCustomWPFGridCell" xml:space="preserve">
+    <value>{0}
+    
+If your application targets a version of .Net Framework before 4.7.1, but is running on a .Net version of 4.7.1 or higher, you may be able to address this issue by taking advantage of accessibility enhancements in .Net 4.7.1. To do this, you will need to enable accessibility switches as described on the following page: https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility#accessibility-switches</value>
```

Comment:
"running on .NET version of 4.7.1 or higher" somehow seems strange to my ear. Would it make sense to add a bit to explain how this could happen, something like "running on a system where .NET version 4.7.1 or higher is installed" ?

Reply:
Done